src/components/RecipePage.vue

```diff
@@ -0,0 +1,302 @@
+    <!-- Summary: Has a title, buttons for actions such as leaving a review -->
+    <!-- Images: A grid of images of the recipe -->
+    <!-- Details: Details such as cook/prep/full time, serving size -->
+    <!-- Ingredients: A list of ingredients used by the recipe -->
+    <!-- Cookware: Pots, pans, optionally listed out if they're detailed in the recipe -->
+    <!-- Steps: Steps required to complete the recipe -->
+<template>
+    <!-- Single root element: Required for Vue -->
+    <div id="recipePage">
+        <!-- Summary -->
+        <div id="recipeSummary">
+            <div id="recipeSummaryTitle">
+                <!-- Title and ratings -->
+                <span class="md-display-1">{{name}}</span>
+                <md-rating-bar style="margin:auto" v-model="rating" :md-max-rating="5" class="md-primary" disabled></md-rating-bar>
+
+                <!-- Reviews | made it | photos -->
+                <md-button class="md-primary">{{reviews.length}} Reviews</md-button>
+                <md-button class="md-primary">{{madeIt}} Made It</md-button>
+                <md-button class="md-primary">{{photos.length}} Photos</md-button>
+            </div>
+
+            <!-- Recipe author and description -->
+            <span class="md-body-2">Recipe by <u>{{submitter}}</u></span><br>
+            <span class="md-subheading">{{description}}</span><br>
+
+            <!-- Save | Made It | Review | Report -->
+            <div id="summaryFlexButtons">
+
+                <!-- Save buttons -->
+                <md-button v-if="!mock.savedIt" class="summaryButtons md-raised md-accent md-dense" v-on:click="toggleSavedIt">Save</md-button>
+                <md-button v-else class="summaryButtons" v-on:click="toggleSavedIt">Saved</md-button>
+
+                <!-- Made it buttons -->
+                <md-button v-if="!mock.madeIt" class="summaryButtons md-raised md-accent md-dense" v-on:click="toggleMadeIt">I Made It</md-button>
+                <md-button v-else class="summaryButtons" v-on:click="toggleMadeIt">Made It</md-button>
+
+                <!-- Review buttons -->
+                <md-button id="reviewButton" class="summaryButtons md-raised md-accent md-dense">Review</md-button>
+
+                <!-- Report buttons -->
+                <md-button class="summaryButtons md-raised md-accent md-dense">Report</md-button>
+            </div>
+        </div>
+
+        <!-- Images -->
+        <div id="recipeImages">
+            <!-- Primary image -->
+            <md-image class="recipePrimaryImage" :md-src="photos[0]" alt="Recipe Image"></md-image>
+
+            <!-- Show smaller previews for non-primary images -->
+            <div id="recipeSecondaryImages">
+                <md-image v-for="photo, key, index in photos" v-if="index < 3" class="recipeSecondaryImage" :md-src="photo" alt="Recipe"></md-image>
+                <md-button id="recipeImageGallery" class="md-raised md-accent" v-else-if="index == 4">+</md-button>
```

Comment:
Maybe not a top priority for this moment but Vuetify has a nice [carousel component](https://vuetifyjs.com/components/carousels) if you wanted to have the gallery button open a modal displaying all the images.